nuxeo-core/nuxeo-core/src/main/resources/schema/coldstorage.xsd

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
+<xs:schema targetNamespace="http://www.nuxeo.org/ecm/schemas/coldstorage/"
+           xmlns:xs="http://www.w3.org/2001/XMLSchema"
+           xmlns:nxs="http://www.nuxeo.org/ecm/schemas/coldstorage/">
+
+  <xs:include schemaLocation="core-types.xsd" />
+
+  <xs:element name="content" type="nxs:content" />
+  <xs:element name="beingRetrieved" type="xs:boolean" />
```

Comment:
Aren't we also adding elements to say for how long the blob will be available ?
And how about adding the infos of the user who requested the blob at the same place ?

Reply:
why not but for moment we should keep as short as possible to keep the things simples and we will iterate over it and we will discuss about theres questions